src/Input/Select/Select.disableTyping.test.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import 'jest-styled-components';
+import '@testing-library/jest-dom/extend-expect';
+import { fireEvent, render, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Select from './Select';
+
+describe('<Select disableTyping> onBlur', () => {
+  it('should call onBlur when close menu', () => {
+    const onBlurSpy = jest.fn();
+    const Component = () => (
```

Comment:
Should this component be made common as these tests are only for checking onBlur is called when disabled typing is true?

Reply:
Updated!